cpp/examples/CMakeLists.txt

```diff
@@ -20,6 +20,18 @@ set_target_properties(BASIC_IO_TEST PROPERTIES INSTALL_RPATH "\$ORIGIN/../../lib
 target_include_directories(BASIC_IO_TEST PRIVATE ../include ${cuFile_INCLUDE_DIRS})
 target_link_libraries(BASIC_IO_TEST PRIVATE kvikio CUDA::cudart)
 
+# Enable supported cuFile features in KvikIO examples
+if(cuFile_FOUND)
```

Comment:
If we end up needing this in another place it would be good to extract this logic, but fine with the duplication for now.